appium/lib/screen-object/accessors/button.rb

```diff
@@ -21,7 +21,7 @@ module ScreenObject
   module AppElements
     class Button < AppElements::Element
 
-      def tap
+      def click
```

Comment:
You shouldn't have renamed it. Onn mobile it is a tap and not click. You can add click as an alias though using `alias_method :tap, :click`